docs/scenario-dashboard.md

```diff
@@ -0,0 +1,228 @@
+local batch processing, and cloud deployment.
+
+
+#### Entry point
+The main entry point is the `run_job` function. You will call this method as a wrapper around your
+own main function. Using this wrapper allows for continuous looped processing (when needed)
+and failure handling.
+
+```python
+from monodrive.jobs import run_job
+
+def main():
+    """main uut driver function"""
+    # ... your code here ...
+
+if __name__ == '__main__':
+    run_job(main, verbose=True)
+```
+
+#### Configuration
+To retrieve the configuration for the current job assignment, you will call the `get_simulator`
+function. Individual config file paths will be parsed from the CLI arguments and pre-loaded into
+the returned `Simulator` object.
+
+```python
+from monodrive.jobs import get_simulator
+
+def main():
+    """main uut driver function"""
+    # ...
+    simulator = get_simulator()
+    # ...
+```
+
+#### Results  
+Finally, at the end of main script, you will create a `Result` object and store it with the
+`set_result` function. These metrics are defined by the user as desired. They will be surfaced
+to the Batch Scenario Manager results page, but are not required for the batch processing to run properly.
+
+```python
+from monodrive.jobs import set_result, Result, ResultMetric
+
+def main():
+    """main uut driver function"""
+
+    # ...
+
+    result = Result()
+    result.pass_result = True
+    result.metrics.append(
+        ResultMetric(
+            name='max_lane_deviation',
+            score=0.123
+        )
+    )
+    set_result(result)
+```
+ 
+#### Example
+For a full working UUT, please check out this [collision avoidance](https://github.com/monoDriveIO/monodrive-python-client/tree/master/examples/collision_avoidance)
+use case, written in Python. This directory includes both a replay and closed loop mode example.
+
+*(C++ example coming soon)*
+
+
+## Configurable Selections
+
+### Batch Scenario Manager Menu
+
+The Batch Scenario Manager Menu shows a summary of all the configurations that have been selected in the configuration selection process: vehicles and sensor configurations, scenario configurations, and weather configurations. On the bottom right corner of the menu, an user may use "Select Configurations" to make and/or edit selections.
+
+  <div class="img_container">
+    <img class='wide_img' src="../imgs/dashboard_starting_menu.png"/>
+  </div>
+
+
+### Vehicles & Sensors
+
+When using closed loop simulation mode, the ego vehicle will be defined in the scenario file with the tag, `ego`. When making vehicle selections, select or add the vehicle(s) that will be used in the closed loop scenario file. The vehicle's `class_path` and `body_color` will be automatically updated in the vehicle object tagged `ego`. For example, `scenario_aebs.json` file in the UUT Python Example under [Configuring UUT](/scenario-dashboard/#connecting-your-code).
```

Comment:
all of these text blocks should have line breaks in them to be readable within a text editor. Right now, it is a single massive line. Markdown rendered will not add a linebreak unless you insert a blank line or leave two spaces at the end of a line